core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1531,21 +1547,19 @@ public static boolean isClassProperty(String key) {
     propertiesByKey = new HashMap<>();
     validPrefixes = new HashSet<>();
     validProperties = new HashSet<>();
+    validTableProperties = new HashSet<>();
 
     for (Property p : Property.values()) {
+      propertiesByKey.put(p.getKey(), p);
       if (p.getType().equals(PropertyType.PREFIX)) {
         validPrefixes.add(p.getKey());
       } else {
+
```

Comment:
Not sure how this made it in. It seems a few blank lines made their way in. It's not a big deal, but the changeset is smaller if these are omitted when not needed.